dbms/src/Storages/StorageDeltaMerge.cpp

```diff
@@ -774,69 +774,7 @@ BlockInputStreams StorageDeltaMerge::read(
 
     auto rs_operator = parseRoughSetFilter(query_info, columns_to_read, context, tracing_logger);
 
-    auto streams = store->read(
-        context,
-        context.getSettingsRef(),
-        columns_to_read,
-        ranges,
-        num_streams,
-        /*max_version=*/mvcc_query_info.read_tso,
-        rs_operator,
-        query_info.req_id,
-        query_info.keep_order,
-        /* is_fast_scan */ query_info.is_fast_scan,
-        max_block_size,
-        parseSegmentSet(select_query.segment_expression_list),
-        extra_table_id_index);
-
-    /// Ensure read_tso info after read.
-    checkReadTso(mvcc_query_info.read_tso, context.getTMTContext(), context, global_context);
-
-    LOG_TRACE(tracing_logger, "[ranges: {}] [streams: {}]", ranges.size(), streams.size());
-
-    return streams;
-}
-
-BlockInputStreams StorageDeltaMerge::read(
```

Comment:
One newbee question:  why the interface can remove?

Reply:
Previously, we use this interface to explicitly send a ScanContextPtr to DeltaMergeStorage. In this PR, I put ScanContextPtr into MvccQueryInfo, so we can reuse the old read interface instead of adding a special interface.
https://github.com/pingcap/tiflash/pull/6472/files#diff-a474c83dcde48b6f38ca42259605a51d8845e89bd9b14cc6309522dd179ed56cL738-R737